client/src/pages/Review/AssessmentsPanel.tsx

```diff
@@ -10,21 +10,34 @@ import divider from "./divider.svg";
 interface AssessmentPanelProp {
   assessments: Assessment[];
   onAssessmentClick(assessment: Assessment, index: number): void;
+  onAssessmentDelete(assessment: Assessment, index: number): void;
 }
 
 const AssessmentsPanel = ({
   assessments,
   onAssessmentClick,
+  onAssessmentDelete,
 }: AssessmentPanelProp) => {
   const renderAssessment = (assessment: Assessment, index: string) => {
     return (
-      <li key={index}>
+      <li key={index} className="flex flex-row group/assessment">
         <AssessmentCard
           assessment={assessment}
           onAssessmentClick={() =>
             onAssessmentClick(assessment, parseInt(index))
           }
         />
+        <Button
+          icon="delete"
+          variant="tonal"
+          ripple={false}
+          className={classnames(
+            "text-sm text-on-primary-container ml-1 overflow-hidden duration-300",
+            "before:bg-primary-95 hover:before:bg-primary-90",
+            "w-0 p-0 opacity-0 group-hover/assessment:w-auto group-hover/assessment:p-2.5 group-hover/assessment:opacity-100"
```

Comment:
```suggestion
            "w-0 p-0 opacity-0 group-hover/assessment:w-auto group-hover/assessment:p-2.5 group-hover/assessment:opacity-100",
            "focus-within/assessment:w-auto focus-within/assessment:p-2.5 focus-within/assessment:opacity-100"
```

Shows the button when it's focused, for better accessibility
![image](https://user-images.githubusercontent.com/70448914/227597964-88a90b4e-e409-4cb2-ab16-98cde1fbff52.png)
****